airbyte-webapp/src/pages/BuilderPage/components/YamlTemplate.ts

```diff
@@ -0,0 +1,46 @@
+// TODO: replace with API call to get starting contents
```

Comment:
I tried this and it gave me this error:
```
ERROR in src/pages/BuilderPage/components/YamlEditor.tsx
  Line 8:1:  Unexpected '!' in '!!raw-loader!./template.yaml'. Do not use import syntax to configure webpack loaders  import/no-webpack-loader-syntax
```

So I tried disabling the eslint rule, which resulted in this error:
```
webpack compiled with 1 error
ERROR in src/pages/BuilderPage/components/YamlEditor.tsx:9:29
TS2307: Cannot find module '!!raw-loader!./template.yaml' or its corresponding type declarations.
     7 |
     8 | // eslint-disable-next-line import/no-webpack-loader-syntax
  >  9 | import defaultTemplate from "!!raw-loader!./template.yaml";
       |                             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    10 |
    11 | export const YamlEditor: React.FC = () => {
    12 |   const [locallyStoredEditorValue, setLocallyStoredEditorValue] = useLocalStorage<string>(
```

I even then tried to add an `index.d.ts` file with the contents `declare module "*.yaml";` in this directory, which fixed the typescript errors. But then when I tried to load the builder in an incognito window, the starting value was 
```
/static/media/template.23913ef2b21a1e799794.yaml
```
instead of the actual yaml file contents, which I'm not even sure what that means (does this mean I would need to asynchronously `fetch` that?)

I wanted to do this import so that I could avoid having asynchronous code just to fetch the starting value of the editor, while keeping this value in a separate file. Also, since this is just temporary until we have an endpoint in the connector-builder-server to return this template, I think keeping it as a typescript file should be okay for now, even though its slightly hacky